lib/jxl/blending.cc

```diff
@@ -361,30 +361,32 @@ Status PerformBlending(
     } else if (ec_blending[i].mode == PatchBlendMode::kBlendAbove) {
       size_t alpha = ec_blending[i].alpha_channel;
```

Comment:
why  x0 is not  used  in line 359 (tmp.Row(3 + i)[x] = bg[3 + i][x] + fg[3 + i][x];)

Reply:
Because that's a mistake ;)